src/Commerce/Generator/PayPal/CLI.php

```diff
@@ -0,0 +1,480 @@
+<?php
+
+use Tribe__Cli__Meta_Keys as Meta_Keys;
+use Tribe__Tickets__Commerce__PayPal__Order as Order;
+use function WP_CLI\Utils\format_items;
+use function WP_CLI\Utils\make_progress_bar;
+
+/**
+ * Class Tribe__Cli__Commerce__Generator__PayPal__CLI
+ *
+ * @since TBD
+ */
+class Tribe__Cli__Commerce__Generator__PayPal__CLI {
+
+	/**
+	 * @var string The current order status, a utility field
+	 */
+	protected $order_status;
+
+	/**
+	 * Generates the PayPal orders for a post.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args
+	 * @param array $assoc_args
+	 *
+	 * @throws \WP_CLI\ExitException
+	 */
+	public function generate_orders( array $args = array(), array $assoc_args = array() ) {
+		$post_id = $this->parse_post_id( $args );
+
+		/** @var \Tribe__Tickets__Commerce__PayPal__Main $paypal */
+		$paypal = tribe( 'tickets.commerce.paypal' );
+
+		$ticket_ids = $this->parse_ticket_ids( $assoc_args, $post_id );
+
+		list( $tickets_min, $tickets_max ) = $this->parse_attendees_min_max( $assoc_args );
+
+		$orders_count = $this->parse_count( $assoc_args );
+		$order_status = $this->parse_order_status( $assoc_args );
+		$post         = get_post( $post_id );
+
+		WP_CLI::log( "Generating PayPal {$orders_count} orders for post {$post_id}" );
+
+		$progress  = make_progress_bar( 'Generating orders', $orders_count );
+		$generated = array();
+
+		$this->hijack_request_flow();
+
+		for ( $k = 0; $k < $orders_count; $k ++ ) {
+			$user_id   = 0;
+			$ticket_id = $ticket_ids[ array_rand( $ticket_ids, 1 ) ];
+			$ticket    = $paypal->get_ticket( $post_id, $ticket_id );
+
+			$this->backup_ticket_total_sales( $ticket_id );
+
+			$ticket_qty = mt_rand( $tickets_min, $tickets_max );
+
+			$inventory  = $ticket->inventory();
+			$inventory  = - 1 === $inventory ? 'unlimited' : $inventory;
+			$ticket_qty = 'unlimited' === $inventory ? $ticket_qty : min( $ticket_qty, (int) $inventory );
+
+			WP_CLI::log( "Generating an order for {$ticket_qty} attendees for ticket {$ticket_id}, current ticket inventory is {$inventory}" );
+
+			if ( $ticket_qty === 0 ) {
+				WP_CLI::log( "Not generating attendees for ticket {$ticket_id} as it ran out of inventory" );
+				continue;
+			}
+
+			$this->order_status = $order_status;
+
+			$progress->tick();
+
+			$faker = Faker\Factory::create();
+
+			$transaction_id = strtoupper( substr( md5( $faker->sentence ), 0, 17 ) );
```

Comment:
Why not use the `$faker->asciify( str_repeat( '*' ,17 ) )`? Same kind of result but "more random".

Reply:
Because that method will use symbols too.